ansible_runner/streaming.py

```diff
@@ -87,8 +87,12 @@ def update_paths(self, kwargs):
 
     def run(self):
         while True:
-            line = self._input.readline()
-            data = json.loads(line)
+            try:
+                line = self._input.readline()
+                data = json.loads(line)
+            except (json.decoder.JSONDecodeError, IOError):
+                self.status_handler({'status': 'error'}, None)
```

Comment:
can we give some more text somewhere that this was a JSON parser issue in the transmission/processing phase? Could we pass the job_explanation key, would that be ok?

Reply:
For this to work `finished_callback` needs to be called, because that sends the eof event. Even if Process gets the error status, it will hang waiting for the eof.

Reply:
You are probably right.